Makefile

```diff
@@ -38,7 +38,7 @@ verify-dependency-bounds-sys: test-set-lower-bounds
 	RUSTFLAGS="${DENY}" cargo build ${MANIFEST_PATH_ARGS} --all-features
 
 clippy:
```

Comment:
You might as well add a clippy task for the sys subdirectory as well. There's a good example of that in the stratisd Makefile.